src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax;
+
+import core.basesyntax.exeption.FruitShopExeption;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.stragegyfactory.StrategyFactory;
+import java.util.List;
+
+public class FruitTransaction {
```

Comment:
should be located in separate package `model`. Should contain only field from description  shouldn't contain any business logic. 
<img width="891" alt="image" src="https://user-images.githubusercontent.com/32339836/222081229-6dd494c8-578a-4f61-9f40-6352e8c1ae3b.png">